src/main/java/spleetwaise/commons/exceptions/SpleetWaiseCommandException.java

```diff
@@ -0,0 +1,14 @@
+package spleetwaise.commons.exceptions;
+
+/**
+ * Represents an exception that occurs during command execution in SpleetWaise.
+ */
+public class SpleetWaiseCommandException extends Exception {
```

Comment:
Is there a need to further abstract `ComandException`? Command exceptions have the same behaviour across `address` and `transaction` right?

Reply:
From a software engineering perspective, the decision to abstract exceptions should be guided by principles like DRY (Don’t Repeat Yourself) and maintainability. If both the address and transaction commands share similar behaviour in their exception handling, it might indeed make sense to introduce a more generic CommandException class. This would allow for more consistent handling and reduce code duplication across different modules.

However, we should also consider the potential for future expansion. The requirements for exception handling are likely to diverge significantly between address and transaction commands in the future, having separate exception classes might provide us with greater flexibility. In this case, we could follow the Open/Closed Principle (OCP), ensuring that the system can be extended without modifying existing code. This should be in line with our initial rationale for creating the `spleetwaise.transaction` packages